pkgs/development/python-modules/iOSbackup/default.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  NSKeyedUnArchiver,
+  pycrypto,
+}:
+
+buildPythonPackage rec {
+  pname = "iOSbackup";
+  version = "0.9.925";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-M1Rakknls/qq3x7ngv5r3823D64N77oazuM2pl+T0co=";
+  };
+
+  dependencies = [
+    pycrypto
+    NSKeyedUnArchiver
+  ];
+
+  pythonImportsCheck = [ "iOSbackup" ];
+
+  meta = {
+    description = "Reads and extracts files from password-encrypted iOS backups";
+    homepage = "https://github.com/avibrazil/iOSbackup";
+    license = lib.licenses.lgpl2;
```

There is no statement about `LGPLv2+` upstream, thus we should assume that it's `LGPL2`-only.

```suggestion
    license = lib.licenses.lgpl2Only;
```